CONTRIBUTING.md

```diff
@@ -102,6 +102,33 @@ Please also see [GitHub
 workflow](https://github.com/open-telemetry/community/blob/main/CONTRIBUTING.md#github-workflow)
 section of general project contributing guide.
 
+### How to Get PRs Merged
+
+A PR is considered to be **ready to merge** when:
+
+* It has received two approvals from Collaborators/Maintainers (at
```

Comment:
The two sentences in this bullet are somewhat contradictory... maybe we revise to be something like:

```suggestion
* Either of the following are true:
  * The PR has received at least two approvals from Collaborators/Maintainers with different employers.
  * The PR has received a single approval from a Maintainer *and* there's a pointer to a public discussion where at least two Collaborators/Maintainers with different employers have agreed upon the approach taken.
```